images/runtime/php-fpm/template.Dockerfile

```diff
@@ -21,5 +21,9 @@ ENV ORYX_AI_INSTRUMENTATION_KEY=${AI_KEY}
 
 COPY --from=startupCmdGen /opt/startupcmdgen/startupcmdgen /opt/startupcmdgen/startupcmdgen
 RUN ln -s /opt/startupcmdgen/startupcmdgen /usr/local/bin/oryx
+RUN if ["%PHP_VERSION%" = "3.7.30"]; then curl -LO http://ftp.de.debian.org/debian/pool/main/a/apache2/apache2-data_2.4.54-2_all.deb \
```

Comment:
Is `"3.7.30"` here a typo? I'm not sure that we support php version `3.7.*`.

Reply:
@pauld-msft, removed these changes. Looks like apt-update and apt-get install apache2 in the dockerfile installs the latest version of apache for us. So, these commands are probably unnecessary.